test/ui-automation/test/helpers/index.js

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+'use strict';
+
+const chapi = require('./chapi');
+const wallet = require('./wallet');
+const issuer = require('./issuer');
+const verifier = require('./verifier');
+
+const api = {};
+module.exports = api;
+
+api.chapi = chapi;
+api.wallet = wallet;
+api.issuer = issuer;
+api.verifier = verifier;
+
+// TODO - get the values from environmental variables
+api.WALLET_NAME = 'wallet.local.trustbloc.dev'
+api.WALLET_URL = 'https://wallet.local.trustbloc.dev'
+api.ISSUER_URL = 'https://issuer.local.trustbloc.dev'
+api.VERIFIER_URL = 'https://rp.local.trustbloc.dev'
```

Comment:
remember, these ones will change for each environment. We might need some config setup for this.

Reply:
yeh, there is a TODO in line 22- https://github.com/trustbloc/sandbox/pull/982/files/3adc9f0fc629594581441b0405f859fd57f2e39e#diff-5394ae9249c1f62e58f03d730f5bcfd8d1eef043b98bbd8354a10b7fd966cc45R22